README.md

```diff
@@ -45,8 +45,17 @@ The app targets shoppers across Australia who are looking for unique and quality
 - Japanese food products lovers
 
 ## User Stories 
+We construct user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying potential users' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+A problem Shizuka faces is the difficulty travelling to supermarkets where authentic Japanese food products are available, even with her three children. With the demands of raisingtrello  her children, she also needs to save time when shopping for her daily food. She is particularly concerned about the security of websites, especially when shopping online, and prefers to use sites that offer a secure login facility.
```

Comment:
This sentence is not very concise, maybe changed from..

`A problem Shizuka faces is the difficulty travelling to supermarkets where authentic Japanese food products are available, even with her three children. With the demands of raisingtrello  her children, she also needs to save time when shopping for her daily food.`

to

"Shizuka, a mom of three, is facing the difficulty of travelling to supermarkets in a daily basis to shop for authentic Japanese food products. She needs a online grocery website that is user-friendly and easy navigated where she can browse and shop online when the kids are taking a nap and get the products delivered to home without the hassle of going to the physical shops."


Also, maybe change this term `secure login facility` to "secure login feature"